mzLib/FlashLFQ/IsoTracker/SearchTarget.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Omics.Modifications;
+
+namespace FlashLFQ.IsoTracker
+{
+    public class SearchTarget
```

Comment:
Consider renaming class to IsoTrackerResultFilter. Or something closer to what the class does.